db/CouriersDB.go

```diff
@@ -15,14 +15,14 @@ func NewCourierPostgres(db *sql.DB) *CourierPostgres {
 
 type Courier struct {
 	IdCourier        uint16 `json:"id_courier"`
```

Comment:
May be it is better to use "Id" instead of "IdCourier"

Reply:
fixed